firmware/util/cli_registry.cpp

```diff
@@ -552,10 +552,7 @@ void handleConsoleLine(char *line) {
 
 	int ret = handleConsoleLineInternal(line, lineLength);
 
-	if (ret == 0) {
-		efiPrintf("unknown command [%s]", line);
-		return;
-	} else if (ret < 0) {
+	if (ret < 0) {
```

Comment:
do we still want to check >0 vs <0 for unknown vs. failed command?

Reply:
Unknown command error reported from `handleConsoleLineInternal` and -1 is returned.
`handleConsoleLine` reports toplevel error if command is failed to handle due to any reason. Exact reason is emitted by `handleConsoleLineInternal` or `handleActionWithParameter`